coordinator/manifest/manifest.go

```diff
@@ -527,6 +529,10 @@ func (m Manifest) Check(zaplogger *zap.Logger) error {
 		}
 	}
 
+	if len(m.RecoveryKeys) < int(m.Config.RecoveryThreshold) {
```

Comment:
```suggestion
	if uint(len(m.RecoveryKeys)) < m.Config.RecoveryThreshold {
```
This way, conversion is always safe, while the other way round it may overflow.